web/forms.py

```
@@ -1517,6 +1519,37 @@ class StudentEnrollmentForm(forms.Form):
     )
 
 
+class MeetupForm(forms.ModelForm):
+    class Meta:
+        model = Meetup
+        fields = ["title", "description", "date", "link", "location", "event_type"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)